src/packages/v2v3/components/V2V3Project/ProjectDashboard/components/NftRewardsPanel/NftReward/PreviewAddRemoveNftButton.tsx

```diff
@@ -3,7 +3,7 @@ import { Trans, t } from '@lingui/macro'
 import { Button } from 'antd'
 import { useCallback } from 'react'
 import { twMerge } from 'tailwind-merge'
-import { emitConfirmationDeletionModal } from '../../../utils/modals'
+import { emitConfirmationDeletionModal } from '../../../../../../../../hooks/emitConfirmationDeletionModal'
```

Comment:
possible to sort out some of these bong import paths? 😅

Reply:
Ah my bad, will fix